packages/AlgernonTests.package/ALGTestAction.class/instance/assertTypeCreateListOfAlgeractions..st

```diff
@@ -0,0 +1,3 @@
+helpers
+assertTypeCreateListOfAlgeractions: aType
```

Comment:
I think this name could be more descriptive. Something like `assertAlgeractionsNotEmptyForType:`